### PR TITLE
Some config cleanup for gke-large-deploy and make it run periodically

### PR DESCRIPTION
**Pull request by shyamjvs**
/cc @wojtek-t

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -6598,18 +6598,17 @@
     "args": [
       "--cluster=gke-large-deploy",
       "--deployment=gke",
-      "--down=false",
       "--extract=ci/latest",
       "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
       "--gcp-node-image=gci",
       "--gcp-project=k8s-scale-testing",
       "--gcp-zone=us-east1-a",
-      "--gke-create-command=container clusters create --quiet --max-nodes-per-pool=1000",
```
Comment (wojtek-t) [inline jobs/config.json]: Let's not remove this

Comment (shyamjvs) [reply jobs/config.json]: It's the default.

Comment (wojtek-t) [reply jobs/config.json]: Aah ok, didn't know that.

Comment (krzyzacy) [reply jobs/config.json]: looks like in kubetest default does not have `--max-nodes-per-pool=1000`? Or that's the default in gke?

File: prow/config.yaml [hunk 2]
```diff
@@ -10319,7 +10319,7 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
-- cron: "0 0 31 2 *" # manual job, set to Feb.31st so it will never be triggered
+- interval: 20m
```
Comment (krzyzacy) [inline prow/config.yaml]: and this becomes a gke-large-updown job essentially?

File: jobs/config.json (hunk 1 above)
Comment (shyamjvs) [reply jobs/config.json]: Yes... The gke documentation says the default value is 1000 - https://cloud.google.com/sdk/gcloud/reference/container/clusters/create

File: prow/config.yaml (hunk 2 above)
Comment (shyamjvs) [reply prow/config.yaml]: Yeah... You could say that.

**Pull request merged**